structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,201 @@
+
+Each community is assigned one single vote. Individual communities are responsible for coming up with their own process for determining their vote, but it must follow these rules:
+
+* The process must be democratic and give a voice to any community member that is otherwise eligible to join the Program's Discord server under the Collaborator role, even if they're not currently present on the server
+* Each eligible member of the community must have equal voting power
+* Eligible members may only be excluded via a reasonable inactivity threshold
+* The process must be formally documented somewhere, but this documentation may be kept internal to the given community
+* Participating communities must allow the Program's Manager team to audit their process, which may be motivated by reports from the given community's members, or from other Program members, including the Managers themselves
+
+Once a community has decided how it's going to vote, it must designate a single eligible community member to represent that vote. This representative must report their community's vote in the attached thread, specifically mentioning the community's name and the option being voted for, and providing proof of the results --- such as a screenshot of an internal poll.
+
+Participating communities also have the option of abstaining from the vote. This must either be reported in the thread as described above or, in the case of a community that wishes to abstain from all votes, directly to a Program Manager in advance. Regardless of the method, documented consensus among eligible community members must be provided.
+
+All polls must include a closing date that is set at least ten days into the future. If needed, this date may be pushed back by up to a month, as long as an informal agreement between the majority of the Program's participating, non-abstaining communities has been reached, and documented within the thread attached to the poll.
+
+At any point before the poll has closed, any Collaborator can use their right to veto, if allowed by the process. This right is described below.
+
+Once the poll has been closed, the vote should be interpreted as described by the process the poll is a part of.
+
+### The right to veto
+
+While a poll is open and accepting votes, any Collaborator or Manager can use their right to veto by sending a message to the attached thread, detailing why they'd like to veto the vote.
+
+If another Collaborator or Manager isn't satisfied with the reasoning behind the veto, they can attempt to overrule the veto. To start this process, a new poll is created to allow voters to decide whether to overrule the veto. This poll lasts for three days and, if more than half of the individual votes are positive, the veto is overruled. If not, the original vote should be treated as if it has failed.
```

Comment:
This section does not adequately clarify what happens after a Collaborator or Manager exercises their veto. How much time do Collaborators or Managers have after such an exercise of veto power to make their objection and start the process to overrule the veto, before the original vote is declared in the negative due to the veto?

Does "voters" in relation to the veto overrule poll here mean the voting communities, whereby the members of that community decide to cast a single vote in accordance to the above section, or does it mean the Collaborators and Managers which each get a single, individual vote?

What happens to the original poll's closing date while a veto overrule poll is open? Is the original poll's remaining duration frozen until the results of the veto overrule poll is finished? If not, does it mean that if a veto overrule poll closes at a time past the original poll's closing date, then the original poll is also immediately closed?

Reply:
1, I'll update the document to state that, if nobody attempts to overrule a veto, it will apply when the original vote closes

2. Whoever is able to vote on the original poll. As vetos happen at an individual (rather than community) level, I think this should be done using individual votes.

3. I'll clarify that a vote to overrule will prevent the original vote from finishing until the vote to overrule has finished.